chromadb/api/types.py

```diff
@@ -189,6 +200,40 @@ class IndexMetadata(TypedDict):
     time_created: float
 
 
+class ChromaMode(str, Enum):
```

Comment:
nit: could we call this OperatingMode or something instead? ChromaMode doesn't feel like it tells me very much